src/eos-dropbox-app

```diff
@@ -121,8 +121,13 @@ class DropboxLauncher():
         self._open_dropbox_directory()
 
     def _name_lost(self, *args):
-        logging.info("Another instance of dropbox is already running")
-        self._already_running = True
+        if not get_dropbox_directory():
+            logging.info("Another instance of dropbox is already running but not "
```

Comment:
"but not Dropbox folder was found" --> "but Dropbox folder was not found"
"dameon" --> "daemon"